classifier_and_summarizer/summarizer/text_summarizer.py

```diff
@@ -0,0 +1,73 @@
+''' Module to perform text summarization'''
+
+from gensim.summarization.summarizer import summarize
+from summarizer import Summarizer
+
+from classifier_and_summarizer.summarizer.incorrect_input import \
+    IncorrectInputError
+
```

Comment:
Looking at the current structure, we have basically 2 implementations of the TextSummarizerInterface, but both the implementations do the same job. This makes the interface kind of redundant. In this case, it looks like the previous structure was better where we had a single class with the 2 summarizers being invoked in different methods of the class. Do you think we can revert back to that structure? Appologies for the repeated effort.

Reply:
That makes sense. Thanks for pointing this out. Will update it accordingly!